plugins/google_drive/help.md

```diff
@@ -1,46 +1,46 @@
 # Description
 
-[Google Drive](https://www.google.com/drive/) is an online file storage and management system. This plugin allows you to upload and retrieve files from Google drive.
-
-This plugin utilizes the [Google Drive API](https://developers.google.com/drive/v3/web/about-sdk).
+Upload and retrieve files from Google Drive
```

Comment:
```suggestion
[Google Drive](https://www.google.com/drive/) is an online file storage and management system. This plugin allows you to upload and retrieve files from Google drive.

This plugin utilizes the [Google Drive API](https://developers.google.com/drive/v3/web/about-sdk).
```

Reply:
I couldn't find a way to use markdown links in plugin.spec.yaml to then generate the help.md file properly. Troubleshooting/Links sections also never get properly populated. But since many plugins already have manually edited help file, I guess we can just follow that route here too.